examples/vortex-mpi.py

```diff
@@ -110,52 +116,55 @@ def main(ctx_factory=cl.create_some_context):
 
     visualizer = make_visualizer(discr, order + 3 if dim == 2 else order)
 
-    initname = initializer.__class__.__name__
-    eosname = eos.__class__.__name__
-    init_message = make_init_message(dim=dim, order=order,
+    extra_init = get_extra_init_euler(
+        cfl=current_cfl,
+        constant_cfl=constant_cfl,
+        initname=initializer.__class__.__name__,
+        eosname=eos.__class__.__name__)
+    init_message = make_init_message(dim=dim, order=order, casename=casename,
                                      nelements=local_nelements,
                                      global_nelements=global_nelements,
                                      dt=current_dt, t_final=t_final, nstatus=nstatus,
-                                     nviz=nviz, cfl=current_cfl,
-                                     constant_cfl=constant_cfl, initname=initname,
-                                     eosname=eosname, casename=casename)
+                                     nviz=nviz, extra_init=extra_init)
```

Comment:
In a way, this is all information that should also be recorded in logpyle's constants, so that these run properties are recorded. Maybe this "init message" thing can be combined with/based on that facility?

cc @matthiasdiener